src/journeys_utils.js

```diff
@@ -360,8 +363,14 @@ function generateIframeOuterCSS() {
 						'transition: all 0' + (journeys_utils.animationSpeed / 1000) + 's ease;';
 	}
 
+	var isCenterOverlay = metadata.type === "center_overlay";
+
+	if (isCenterOverlay) {
+		document.body.style.background = "rgba(0,0,0,0.4)";
```

Comment:
Could be in the future, that the customer whats to change the overlay color, from shade gray to something special. Shade gree or like that. But for now it's ok to do it in this way.